QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,91 @@
+
+There is no specific implementation/guidelines stated as to how authors are maintaining that library.
+
+## Documentation
+
+`libconsole-bridge-dev` does not provide any type of documentation of how the library is used in their source code repository. There is a [Wiki](http://wiki.ros.org/console_bridge) related to their functionality and expected behavior for its old ROS package.
+
+Feature and API documentation are not addressed.
+
+### License and Copyright Statements
+
+It is not explicitly stated a License or Copyright statement for the library. There is no particular LICENSE file in the source files or on the repository of the library. However, the source files (.cpp and .hpp) all contain the following BSD license header: 
+
+	/*********************************************************************
+	* Software License Agreement (BSD License)
+	* *
+	* * Copyright (c) 2008, Willow Garage, Inc.
+	* * All rights reserved.
+	* *
+	* * Redistribution and use in source and binary forms, with or without
+	* * modification, are permitted provided that the following conditions
+	* * are met:
+	*
+	* * Redistributions of source code must retain the above copyright
+	* notice, this list of conditions and the following disclaimer.
+	* * Redistributions in binary form must reproduce the above
+	* copyright notice, this list of conditions and the following
+	* disclaimer in the documentation and/or other materials provided
+	* with the distribution.
+	* * Neither the name of the Willow Garage nor the names of its
+	* contributors may be used to endorse or promote products derived
+	* from this software without specific prior written permission.
+	*
+	* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+	* "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+	* LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+	* FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+	* COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+	* INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+	* BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	* LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+	* CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+	* LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+	* ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+	* POSSIBILITY OF SUCH DAMAGE.
+	*********************************************************************/
+
+## Testing
+
+The `libconsole-bridge-dev` library includes a simple testing to check if it compiles properly the basic simple case use. Does not include any kind of performance tests, coverage information nor static analysis.
+
+## Dependencies
+
+There is no dependency on external sources for the `libconsole-bridge-dev` library, only C++ libraries included in the standard library (mutex, iostream).
+
+## Platform Support
```

Comment:
Maybe for external libraries which are not really supported for specific operative systems it can be considered changing this requirement for a metric more related to "Portability"